main.py

```diff
@@ -131,6 +131,61 @@ def calculate_interest(self, rate: float) -> None:
         print("Debit accounts do not earn interest.")
 
 
+class SavingsAccount:
```

Comment:
`SavingsAccount` seems to be an another account type. It should probably inherit from `Account`, especially because `Customer` expects an account of `Account` type.